src/utils.js

```diff
@@ -60,3 +62,26 @@ export function recalculateActive (terminalsSet, active) {
 		t.emitter.emit('did-change-title')
 	})
 }
+
+// finds the default shell start commmand
+export function getShellStartCommand () {
+	let shellStartCommand
+	if (process.platform === 'win32') {
+		// Windows
+		try {
+			childProcess.spawn('pwsh.exe', ['-v'])
```

Comment:
This will never throw an error since it is not a synchronous function

Reply:
I'm not sure this is worth possibly two calls blocking the main thread every time Atom starts to figure out the default to a config value even after the user changes it.

Reply:
> This will never throw an error since it is not a synchronous function

It throws errors. Try changing it to `pwsh1` and you will see it.

> I'm not sure this is worth possibly two calls blocking the main thread every time Atom starts to figure out the default to a config value even after the user changes it.

The time it takes to spawn these is much less than the speed up we gain from #109.

Reply:
> It throws errors. Try changing it to pwsh1 and you will see it.

it notified me of an error but did not go into the catch clause.

Getting the config for atom is synchronous so async calls to get the default won't work.

> The time it takes to spawn these is much less than the speed-up we gain from #109.

not worth the extra development time.

Reply:
It looks like an async config default is not possible since Atom loads the `config` property of packages synchronously

https://github.com/atom/atom/blob/4a9d56f52cab00a22744858d4077600d5365197a/src/package.js#L262-L273

Reply:
If that's the case, that is a good place to improve Atom in. We need to make loading packages and their configuration async.